.github/workflows/publish-docker-images.yml

```
@@ -395,6 +395,42 @@ jobs:
           cache-from: type=gha
           cache-to: type=gha,mode=max
 
+  db_migration:
+    name: Build DB Migration Job
+    environment: Deployment
+    defaults:
+      run:
+        working-directory: ./
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)